src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int HEADER_LINE_INDEX = 0;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = HEADER_LINE_INDEX + 1; i < data.size(); i++) {
+            String[] parts = data.get(i).split(",");
+
+            if (parts.length >= 3) {
```

Comment:
This check for the number of parts assumes there will always be exactly three parts. Consider checking if parts.length is not less than the expected number of parts to make the code more robust in case the data format changes in the future.